src/Config/ObjectModel/GraphQLRuntimeOptions.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Azure.DataApiBuilder.Config.ObjectModel;
 
-public record GraphQLRuntimeOptions(bool Enabled = true, string Path = GraphQLRuntimeOptions.DEFAULT_PATH, bool AllowIntrospection = true)
+public record GraphQLRuntimeOptions(bool Enabled = true, string Path = GraphQLRuntimeOptions.DEFAULT_PATH, bool AllowIntrospection = true, NestedMutationOptions? NestedMutationOptions = null)
```

Comment:
nit: please split into multiple lines - the constructor parameters have become far too many